crates/core/src/lib.rs

```diff
@@ -12,7 +12,3 @@ pub mod repository;
 #[cfg(feature = "gui")]
 pub mod theme;
 pub mod utility;
-
-use crate::error::ClientError;
-
-pub type Result<T> = std::result::Result<T, ClientError>;
```

Comment:
Nice we got rid of this.

Reply:
yup! We have a few different error types now